external-crates/move/crates/move-compiler/tests/move_check/parser/hexstring_token_eof2.exp

```diff
@@ -1,3 +1,9 @@
+error[E02004]: invalid 'module' declaration
+  ┌─ tests/move_check/parser/hexstring_token_eof2.move:1:8
+  │
+1 │ module M {
```

Comment:
We should probably fix some of these so that they are otherwise valid (besides the thing they are testing)